SwiftAppTemplate/SwiftAppTemplate/APIClient.swift

```diff
@@ -23,27 +24,82 @@ class APIClient {
      fetchConnectionToken once you have StripeTerminal installed in the app.
      */
 
-    static var backendUrl: String = ""
+    static let backendUrl: String = ""
 
+    static let defaultError = NSError(domain: "APIClient Error",
+                                      code: 1,
+                                      userInfo: [
+                                        NSLocalizedDescriptionKey: "APIClient request failed.",
+    ])
+
+//    func urlWithPathComponents(_ pathComponents: [String]) -> URL {
+//        guard let backendURL = URL(string: APIClient.backendUrl) else {
+//            fatalError()
+//        }
+//        var responseURL = backendURL
+//        for component in pathComponents {
+//            responseURL = backendURL.appendingPathComponent(component)
+//        }
+//        return responseURL
+//    }
+//
 //    func fetchConnectionToken(_ completion: @escaping ConnectionTokenCompletionBlock) {
-//        if let backendUrl = URL(string: APIClient.backendUrl) {
-//            let url = backendUrl.appendingPathComponent("connection_token")
-//            let params: [String: Any] = [:] // whatever parameters necessary for your backend to authenticate the client
-//
-//
-//            AF.request(url, method: .post, parameters: params)
-//                .responseJSON { responseJSON in
-//                    switch responseJSON.result {
-//                    case .success(let json):
-//                        if let json = json as? [String: AnyObject],
-//                            let secret = json["secret"] as? String {
-//                            completion(secret, nil)
-//                        }
-//                    case .failure(let error):
-//                        completion(nil, error)
+//        let url = self.urlWithPathComponents(["connection_token"])
+//        let params: [String: Any] = [:] // whatever parameters necessary for your backend to authenticate the client
+//
+//        AF.request(url, method: .post, parameters: params)
+//            .responseJSON { responseJSON in
+//                switch responseJSON.result {
+//                case .success(let json):
+//                    if let json = json as? [String: AnyObject],
+//                        let secret = json["secret"] as? String {
+//                        completion(secret, nil)
 //                    }
-//            }
+//                case .failure(let error):
+//                    completion(nil, error)
+//                }
 //        }
 //    }
-
+//
+//
+//    func createPaymentIntent(_ paymentIntentParams: PaymentIntentParameters,
```

Comment:
We don't need this for WisePad 3 since they can use the SDK provided createPaymentIntent